docs/nightly/en/user-guide/operations/deploy-on-kubernetes/deploy-greptimedb-cluster.md

```diff
@@ -0,0 +1,105 @@
+# Deploy GreptimeDB Cluster
+
+Before the deployment,
+make sure you have already installed [GreptimeDB Operator](greptimedb-operator.md) on your Kubernetes cluster.
+
+## Using Helm for deployment
+
+### Create an etcd cluster
+
+First, establish an etcd cluster to support GreptimeDB by executing the following command:
+
+```shell
+helm install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
+  --set replicaCount=3 \
+  --set auth.rbac.create=false \
+  --set auth.rbac.token.enabled=false \
+  --create-namespace \
+  -n etcd
+```
+
+After the installation,
+you can get the etcd cluster endpoints `etcd.etcd.svc.cluster.local:2379` from the installation logs.
+The endpoints are required for deploying the GreptimeDB cluster in the subsequent step.
+
+### Create a GretpimeDB cluster
+
+Deploy the GreptimeDB cluster, ensuring it connects to the previously established etcd cluster:
+  
+```shell
+helm install greptimedb greptime/greptimedb-cluster \
+  --set meta.etcdEndpoints=etcd.etcd.svc.cluster.local:2379 \
+  --create-namespace \
+  -n greptimedb-cluster
+```
+
+### Setting Resource requests and limits
+
+The GreptimeDB Helm charts enable you to specify resource requests and limits for each component within your deployment.
+
+Here's how you can configure these settings:
+
+```shell
+helm install greptimedb greptime/greptimedb-cluster \
+  --set meta.etcdEndpoints=etcd.etcd.svc.cluster.local:2379 \
+  --set meta.podTemplate.main.resources.requests.cpu=<cpu-resource> \
+  --set meta.podTemplate.main.resources.requests.memory=<mem-resource> \
+  --set datanode.podTemplate.main.resources.requests.cpu=<cpu-resource> \
+  --set datanode.podTemplate.main.resources.requests.memory=<mem-resource> \
+  --set frontend.podTemplate.main.resources.requests.cpu=<cpu-resource> \
+  --set frontend.podTemplate.main.resources.requests.memory=<mem-resource> \
+  --create-namespace \
+  -n greptimedb-cluster
+```
+
```

Comment:
It's better to give a setup example to copy and run with the smallest resource limits.

Reply:
Giving a link to the capacity plan would be good.

Reply:
The details of capacity plan for GreptimeDB Cluster will be written in the next PR.